configs/recognition/tsn/tsn_r50_320p_1x1x8_150e_activitynet_clip_flow.py

```diff
@@ -80,6 +80,7 @@
         filename_tmpl='flow_{}_{:05d}.jpg',
         with_offset=True,
         modality='Flow',
+        start_index=0,
```

Comment:
i thought we moved `start_index` to dataset? @dreamerlin

Reply:
Yes. Move it to [here](https://github.com/open-mmlab/mmaction2/blob/master/mmaction/datasets/base.py#L51) and the [old one](https://github.com/open-mmlab/mmaction2/blob/master/mmaction/datasets/pipelines/loading.py#L62-L65) has been deprecated

Reply:
a little confused, I think I have written start_index in dataset config.

Reply:
oh right my mistake